src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+
+    private static final SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
+    }
```

Comment:
Remove the default constructor. HibernateUtil is a utility class that should not be instantiated, hence a private constructor is needed to enforce non-instantiability.